docs/tutorials/connectomics/mcc.ipynb

```diff
@@ -273,21 +273,24 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "vertex_pvals = []\n",
+    "with warnings.catch_warnings():\n",
+    "    warnings.filterwarnings(\"ignore\", module=\"hyppo\")\n",
+    "    warnings.filterwarnings(\"ignore\", category=RuntimeWarning)\n",
+    "    vertex_pvals = []\n",
     "\n",
-    "for vertex_i in range(n_vertices):\n",
+    "    for vertex_i in range(n_vertices):\n",
     "    \n",
     "    # Get the embedding of the i-th vertex for \n",
-    "    samples = [\n",
-    "        omni_embedding[mice.labels==genotype, vertex_i, :] \n",
-    "        for genotype in np.unique(mice.labels)\n",
-    "    ]\n",
+    "        samples = [\n",
+    "            omni_embedding[mice.labels==genotype, vertex_i, :] \n",
+    "            for genotype in np.unique(mice.labels)\n",
+    "        ]\n",
     "    \n",
     "    # Calculate the p-value for the i-th vertex\n",
-    "    statistic, pvalue, _ = KSample(\"MGC\").test(\n",
-    "        *samples, reps=250, workers=-1\n",
-    "    )\n",
-    "    vertex_pvals.append([vertex_i, statistic, pvalue])"
+    "        statistic, pvalue, _ = KSample(\"MGC\").test(\n",
+    "            *samples, reps=250, workers=-1\n",
+    "        )\n",
+    "        vertex_pvals.append([vertex_i, statistic, pvalue])"
```

Comment:
are these tabs intentional?

Reply:
yes @loftusa these tabs are necessary as this particula code was giving out huge no of warnings that were runtime and by hyppo library so these had to be suppressed.
this is the modified code
`with warnings.catch_warnings():
    warnings.filterwarnings("ignore", module="hyppo")
    warnings.filterwarnings("ignore", category=RuntimeWarning)
    vertex_pvals = []

    for vertex_i in range(n_vertices):
    
    # Get the embedding of the i-th vertex for 
        samples = [
            omni_embedding[mice.labels==genotype, vertex_i, :] 
            for genotype in np.unique(mice.labels)
        ]
    
    # Calculate the p-value for the i-th vertex
        statistic, pvalue, _ = KSample("MGC").test(
            *samples, reps=250, workers=-1
        )
        vertex_pvals.append([vertex_i, statistic, pvalue])`

while this being the original code 
`vertex_pvals = []

for vertex_i in range(n_vertices):
    
    # Get the embedding of the i-th vertex for 
    samples = [
        omni_embedding[mice.labels==genotype, vertex_i, :] 
        for genotype in np.unique(mice.labels)
    ]
    
    # Calculate the p-value for the i-th vertex
    statistic, pvalue, _ = KSample("MGC").test(
        *samples, reps=250, workers=-1
    )
    vertex_pvals.append([vertex_i, statistic, pvalue])`

and as you can see the original code has been put under a function to catch the warnings and then ignore them hence adding indentation was necessary.

Reply:
@rajpratyush Oh cool, didn't see the `with` statement. Seems fine to me!